playbook/app/pb_kits/playbook/pb_multi_level_select/_multi_level_select.tsx

```diff
@@ -122,14 +133,20 @@ const MultiLevelSelect = (props: MultiLevelSelectProps) => {
       if (item.id != id) item.children = modifyValue(id, item.children, check)
       else {
         item.checked = check
-        item.children = modifyRecursive(item.children, check)
+
+        if (variant === "single") {
+          // Single select: no children should be checked
+          item.children = modifyRecursive(item.children, !check)
```

Comment:
Previously `modifyValue` checks the children - but we don't want that. A radio selection only has one selection.